runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpState.java

```diff
@@ -131,32 +129,28 @@ void updateOverall(long currentXp)
 		}
 	}
 
-	private double getCombatXPModifier(Skill skill)
-	{
-		if (skill == Skill.HITPOINTS)
-		{
-			return SHARED_XP_MODIFIER;
-		}
-
-		return DEFAULT_XP_MODIFIER;
-	}
-
 	/**
 	 * Updates skill with average actions based on currently interacted NPC.
 	 *
 	 * @param skill     experience gained skill
 	 * @param npc       currently interacted NPC
 	 * @param npcHealth health of currently interacted NPC
 	 */
-	void updateNpcExperience(Skill skill, NPC npc, Integer npcHealth, int xpModifier)
+	void updateNpcExperience(
+		Skill skill,
+		NPC npc,
+		Integer npcHealth,
+		int worldXpModifier,
+		double combatXpModifier)
```

Comment:
Can we add these to the method's javadoc? I know it was missing `xpModifier` before, but let's not let it drift even further from the real method signature.

Reply:
+1 [commit](https://github.com/runelite/runelite/pull/16910/commits/99be717e00ffbcae6e9cbd16c2ac55d92fa87947?w=1)